Terminal.Gui/Views/TextField.cs

```diff
@@ -147,11 +147,10 @@ public override bool OnLeave (View view)
 				}
 				TextChanged?.Invoke (oldText);
 
-				if (point > text.Count)
-					point = Math.Max (DisplaySize (text, 0) - 1, 0);
+				if (point > text.Count) {
+					point = Math.Max (DisplaySize (text, 0).Item1 - 1, 0);
```

Comment:
Should be `DisplaySize (...).Length` in order for code to be more readable.

Reply:
In this case Item1 is `Size `and not `Length`. I'll change to `DisplaySize (...).Size`.

Reply:
@tig it's not possible to use the named items. It's only possible when we instantiate a new one. Please review that.